scripts/signoff-git-hook.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
```

Comment:
The contents could be imbedded in the `vdev/src/commands/install_git_hooks.rs` file, I'm not sure if that's entirely better.

Reply:
I thought about this too, I think it would help people understand how the files are connected / the logic is all tied in rather than have to traverse two different directories and look at two different places. That was my main motivation for adding the clarifying comment "Used by vdev/src/commands/install_git_hooks.rs" since one of our discussion topics before was increasing readability/documentation. I do prefer merging this script inside the Rust file.

Reply:
I'm having trouble seeing what the style it is I should follow through the link you sent in, is there something I can run like cargo fmt or cargo vdev fmt to make my imports conform to the standard?